app/views/news_tematica/news_tematicas/elegir_contenidos.html.haml

```diff
@@ -36,10 +41,13 @@
   &nbsp; &nbsp;
   = submit_tag('Generar HTML', class: 'boton_reborde_naranja_grande')
 
-:javascript
-  jQuery(function($) {
-    $('.js_ver_resto_titulares').click(function() {
-      $('.js_resto_titulares').slideDown();
-      $(this).hide();
-    });
-  });
+:coffee
```

Comment:
Do you think that the JS should go on the view, or it should better go in a separate .js file? Yesterday I was touching some views which also had javascript, and it seems me a strange... we never include css on the HAML, why including js?  @gponsu , you're the frontender, what do you think?